googlemock/test/gmock-generated-matchers_test.cc

```diff
@@ -120,21 +120,21 @@ TEST(ArgsTest, AcceptsOneTemplateArg) {
 }
 
 TEST(ArgsTest, AcceptsTwoTemplateArgs) {
-  const tuple<short, int, long> t(4, 5, 6L);  // NOLINT
+  const tuple<short, int, long> t(static_cast<short>(4), 5, 6L);  // NOLINT
```

Comment:
This line produces warning `tuple(165): warning C4244: 'initializing': conversion from 'int' to 'short', possible loss of data` and because of `-WX` compiler produce errors for warnings

Reply:
Same bellow

Reply:
The same change was proposed in #1136